gcc_utils.py

```diff
@@ -24,7 +24,7 @@ def get_google_cloud_storage_blob(gcc_bucket, gcc_file_path):
     return blob
 
 
-def gccPrint(*message, severity="DEFAULT"):
+def gccPrint(message, severity="DEFAULT"):
```

Comment:
Breaks other python files using this function,  it doesn't affect readability or the json print on gcp with multiple args
print() has the same multiple args definition fwiw